packages/ruleset/src/rules/property-case-collision.js

```diff
@@ -0,0 +1,15 @@
+const { oas2, oas3 } = require('@stoplight/spectral-formats');
+const { propertyCaseCollision } = require('../functions');
+const { schemas } = require('../collections');
+
+module.exports = {
+  description: 'Property names should not be identical',
```

Comment:
Since the actual message printed for the user is coming from the error, this can also be more descriptive about the rule itself. For context, the purpose of the rule is to prevent tooling from generating colliding property names after converting to a consistent case convention